files/zh-cn/web/html/element/colgroup/index.md

```diff
@@ -1,112 +1,215 @@
+## 使用说明
 
-- `char` {{deprecated_inline}}
+- `<colgroup>` 应出现在 {{HTMLElement("table")}} 内，任何 {{HTMLElement("caption")}} 元素（如果使用）之后，但在任何 {{HTMLElement("thead")}}、{{HTMLElement("tbody")}}、{{HTMLElement("tfoot")}} 和 {{HTMLElement("tr")}} 元素之前。
+- 只有数量有限的 CSS 属性会影响 `<colgroup>`：
+  - {{cssxref("background")}}：各种 `background` 属性将为列组中的单元格设置背景。由于列组的背景色绘制在表格顶部，而背景色绘制在列（{{htmlelement("col")}}）和行组（{{htmlelement("thead")}}、{{htmlelement("tbody")}} 和 {{htmlelement("tfoot")}}）上。只有在表格列组上绘制的每个图层都具有透明背景时，应用于表格列组的背景才会可见。
+  - {{cssxref("border")}}：适用各种 `border` 属性，但前提是 `<table>` 已设置 {{cssxref("border-collapse", "border-collapse: collapse")}}。
+  - {{cssxref("visibility")}}：如果为列组设置 `collapse` 值，则该列组中列的所有单元格都不会被渲染，而跨入其他列的单元格则会被剪切。列组中的这些列原本占据的空间会被移除。但是，其他列的大小仍然按照列组中被折叠列的单元格计算。`visibility` 的其他值没有影响。
+  - {{cssxref("width")}}：`width` 属性定义了列组中的列的最小宽度，如同设置了 {{cssxref("min-width")}}。
 
-  - : This attribute specifies the alignment of the content in a column group to a character. Typical values for this include a period (.) when attempting to align numbers or monetary values. If [`align`](/zh-CN/docs/Web/HTML/Element/colgroup#align) is not set to `char`, this attribute is ignored, though it will still be used as the default value for the [`align`](/zh-CN/docs/Web/HTML/Element/col#align) of the {{HTMLElement("col")}} which are members of this column group.
+## 示例
 
-    > **备注：** Do not use this attribute as it is obsolete (and not supported) in the latest standard. To achieve the same effect as the [`char`](/zh-CN/docs/Web/HTML/Element/colgroup#char), in CSS3, you can use the character set using the [`char`](/zh-CN/docs/Web/HTML/Element/colgroup#char) attribute as the value of the {{cssxref("text-align")}} property.
+请参阅 {{HTMLElement("table")}} 获取介绍通用标准和最佳实践的完整表格示例。
 
-- `charoff` {{deprecated_inline}}
+本示例演示了一个七列表格，分为两个跨多列的 `<colgroup>` 元素。
 
-  - : This attribute is used to indicate the number of characters to offset the column data from the alignment character specified by the **char** attribute.
+### HTML
 
-    > **备注：** Do not use this attribute as it is obsolete (and not supported) in the latest standard.
+两个 `<colgroup>` 元素用于通过创建列组来构建基本表格。每个列组中的列数由 [`span`](#span) 属性指定。
 
-- `span`
+```html
+<table>
+  <caption>
+    个人每周活动
+  </caption>
+  <colgroup>
+    <col />
+    <col span="5" class="weekdays" />
+    <col span="2" class="weekend" />
+  </colgroup>
+  <tr>
+    <th>时段</th>
+    <th>周一</th>
+    <th>周二</th>
+    <th>周三</th>
+    <th>周四</th>
+    <th>周五</th>
+    <th>周六</th>
+    <th>周日</th>
+  </tr>
+  <tr>
+    <th>上午</th>
+    <td>打扫房间</td>
+    <td>足球训练</td>
+    <td>舞蹈课</td>
+    <td>历史课</td>
+    <td>买饮料</td>
+    <td>自习</td>
+    <td>自由时间</td>
+  </tr>
+  <tr>
+    <th>下午</th>
+    <td>瑜伽</td>
+    <td>棋类俱乐部</td>
+    <td>见朋友</td>
+    <td>体操</td>
+    <td>生日派对</td>
+    <td>钓鱼之旅</td>
+    <td>自由时间</td>
+  </tr>
+</table>
```

Comment:
I think the change you made, that is, adding the `th` in the `tr` elemment, is better. Going to update the upstream?

Reply:
oops, I've copied them from `<col>` examples...

I would update them to match the original document.